chat.revolt.RevoltDesktop.yaml

```
@@ -1,16 +1,13 @@
 app-id: chat.revolt.RevoltDesktop
 runtime: org.freedesktop.Platform
-runtime-version: '21.08'
+runtime-version: '22.08'
 sdk: org.freedesktop.Sdk
-
 base: org.electronjs.Electron2.BaseApp
-base-version: '21.08'
-
+base-version: '22.08'
 sdk-extensions:
 - org.freedesktop.Sdk.Extension.node16
```

Can node be bumped to 18?

```suggestion
- org.freedesktop.Sdk.Extension.node18
```

---

done